src/helpers/a11yAPI.ts

```diff
@@ -19,6 +47,10 @@ type QueryAllReturn = Array<ReactTestInstance>;
 type FindReturn = Promise<GetReturn>;
 type FindAllReturn = Promise<GetAllReturn>;
 
+export type QueryOptions = {
```

Comment:
This type should be renamed to `ByRoleOptions` bacause this type of option is only relevant to `byRole` queries.